src/ImageRunModal.jsx

```diff
@@ -273,6 +305,13 @@ export class ImageRunModal extends React.Component {
                                  onChange={value => this.onValueChanged('publish', value)}
                                  default={{ containerPort: null, hostPort: null, protocol: 'TCP' }}
                                  itemcomponent={ <PublishPort />} />
+
+                <label className='control-label' htmlFor='run-image-dialog-env'>{ _("Environment") }</label>
+                <DynamicListForm id='run-image-dialog-env'
+                                 formclass='env-form'
```

Comment:
I tend to not use classes that don't introduce any css but are just used for css selectors in the test. IDs are better fit for this purpose. But it's not important to block this PR from being merged.

Reply:
From my POV it's mostly a matter of taste. Sometimes classes are preferrable if the thing you select is not unique. The one thing to avoid is to attach CSS to IDs.